File: lista_de_exercicios/questao13/questao13.py
class Matematica:
    @staticmethod
    def fatorial(n):
        if n < 0:
            return None
        if n == 0 or n == 1:
            return 1
        resultado = 1
        for i in range(2, n + 1):
            resultado *= i
        return resultado

print(Matematica.fatorial(5))

File: lista_de_exercicios/questao13/questao13.go
package main

import "fmt"

type Matematica struct{}

func Fatorial(n int) int {
	if n < 0 {
		return -1
	}
	if n == 0 || n == 1 {
		return 1
	}
	resultado := 1
	for i := 2; i <= n; i++ {
		resultado *= i
	}
	return resultado
}

func main() {
	fmt.Println(Fatorial(5))
}
